roxie/ccd/ccdmain.cpp

```diff
@@ -717,6 +717,20 @@ int STARTQUERY_API start_query(int argc, const char *argv[])
         if (udpSnifferEnabled && !roxieMulticastEnabled)
             DBGLOG("WARNING: ignoring udpSnifferEnabled setting as multicast not enabled");
 
+        int ttlTmp = topology->getPropInt("@multicastTTL", 1);
+        if (ttlTmp < 0)
+        {
+            multicastTTL = 1;
+            WARNLOG("multicastTTL value (%d) invalid, must be >=0, resetting to default", ttlTmp);
+        }
+        else if (multicastTTL > 10)
```

Comment:
10 was just my judgement on an acceptable upper limit for MC.  I could make it 255 which is the TTL max for udp I think, but that is awfully high for MC.